.github/workflows/release.yml

```diff
@@ -0,0 +1,164 @@
+name: Release new version of Optable SDK and demos
+on:
+  push:
+    tags:
+      - "v.*"
+
+jobs:
+  tests-prettier:
+    uses: ./.github/workflows/reusable-lint-test.yml
+
+  build:
+    needs: [tests-prettier]
+    uses: ./.github/workflows/reusable-build.yml
+
+  deploy-sdk-to-npm:
+    needs: [build]
+    runs-on: ubuntu-latest
+    env:
+      gcp-project-id: 118585658141
+      service-account: gh-ci-optable-web-sdk
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download web artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: browser/dist
+
+      - name: Download lib artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: lib/dist
+
+        # Step will fail if the version is invalid, github.ref_name is the tag name (v.*.*.*)
+      - name: Patch version
+        run: ./scripts/patch-version.sh "${{ github.ref_name }}"
+
+      - name: Setup registry access
+        run: |
+          echo '@optable:registry=https://registry.npmjs.org/' > ~/.npmrc
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.GITHUB_TOKEN }}" >> ~/.npmrc
+
+      - name: Publish to NPM
+        run: npm publish --access public
+
+  define-gcs-versions-to-update:
+    needs: [deploy-sdk-to-npm]
+    runs-on: ubuntu-latest
+
+    outputs:
+      sdk-versions: ${{ steps.sdk-versions.outputs.versions }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install asdf CLI
+        uses: asdf-vm/actions/install@v3
+
+      # We need to determine which versions to update based on the new version
+      # - In case of a pre-release, only update vX.Y.Z-{prerel}
+      # - If not a pre-release
+      #   - Always update vX.Y.Z
+      #   - Always update vX.Y
+      #   - Update latest if the new release version is greater than current
+      #   - Update vX if patching a version for which the minor is the latest, examples:
+      #     - v.12.2.1 and patching v.12.1.4, we don't update v12
+      #     - v.12.2.1 and patching v.12.2.2, we update v12
+      - name: Determine versions to update
+        id: sdk-versions
+        run: ./scripts/versions-to-update.sh @optable/web-sdk ${{ github.ref_name }} >> "$GITHUB_OUTPUT"
+
+      - name: Show versions that will be updated
+        run: |
+          echo "SDK versions: ${{ steps.sdk-versions.outputs.versions }}"
+
+  deploy-sdk-to-gcs:
+    needs: [define-gcs-versions-to-update]
+    strategy:
+      matrix:
+        sdk-version: ${{ fromJSON(needs.define-gcs-versions-to-update.outputs.sdk-versions) }}
+    runs-on: ubuntu-latest
```

Comment:
same here: We should use the 2-core-ubuntu since it is included in our plan. I believe those are small jobs that won't need a more powerful one. Same to all the other runs-on with ubuntu-latest ;)